static/src/javascripts/projects/common/modules/video/video-container.js

```diff
@@ -36,6 +44,7 @@ define([
         $('.video-playlist__item--active').removeClass('video-playlist__item--active');
 
         if (direction === 'next') {
+            fetchImage(containerPos + preloadImageCount);
```

Comment:
Na, just do it at the same time... on a slow connection pressing next, then waiting (with no load), to then be scrolled, I would be pissed. Probably best to add a loading image, but I can do that after.

If we went with the next track paradigm, you could definitely do loading, but you would actually be loading the video, which to me would make sense and actually eradicate the need for holding images.